servant/src/Servant/API/Empty.hs

```diff
@@ -0,0 +1,10 @@
+{-# LANGUAGE DeriveDataTypeable #-}
+{-# OPTIONS_HADDOCK not-home    #-}
+module Servant.API.Empty(EmptyAPI(..)) where
+
+import           Data.Typeable    (Typeable)
+import           Prelude ()
+import           Prelude.Compat
+
+-- | An empty API: one which serves nothing.
```

Comment:
@kosmikus somwhere, e.g in `docs` and `servant-swagger` it will be actually a unit.

I want something to guide the implementors of other interpretations of API types to treat `EmptyAPI` as unit as possible.